python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -399,12 +399,12 @@ def _create_node(self, node_config, tags, count):
                 break
             except botocore.exceptions.ClientError as exc:
                 if attempt == BOTO_CREATE_MAX_RETRIES:
-                    # todo: err msg
                     cli_logger.abort(
-                        "Failed to launch instances. Max attempts exceeded.")
-                    raise exc
+                        "Failed to launch instances. Max attempts exceeded.",
+                        exc=exc,
+                    )
                 else:
-                    cli_logger.print(
+                    cli_logger.warning(
```

Comment:
this is a bit scarier than necessary -- can we revert?

Reply:
oh, I thought "warning" was a more appropriate level according to Python conventions:
![image](https://user-images.githubusercontent.com/4212216/117247610-6d756380-adf3-11eb-9450-c0faee3763e1.png)

But if it's too severe in the Ray logging context we can certainly revert - I'll let you make the call

Reply:
ah well tests passed and this was more of a nit, so we'll keep the change :)